astropy/utils/masked/tests/test_functions.py

```diff
@@ -5,10 +5,11 @@
 coverage.  Complete coverage of all numpy functions is done
 with less detailed tests in test_function_helpers.
 """
+import erfa
```

Comment:
Is there a reason I'm missing why existing tests used the `erfa.ufunc` namespace instead of `erfa`, that doesn't apply to your new tests ?